android/build.gradle

```diff
@@ -1,27 +1,10 @@
-
-buildscript {
-    repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
-    }
-}
-
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion rootProject.ext.compileSdkVersion
```

Comment:
I think it is better if we have an utility function like this:
```
def safeExtGet(prop, fallback) {
    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
}
```
Then: 
```
compileSdkVersion safeExtGet('compileSdkVersion', 23)
buildToolsVersion safeExtGet('buildToolsVersion', '23.0.1')
minSdkVersion safeExtGet('minSdkVersion', 16)
targetSdkVersion safeExtGet('targetSdkVersion', 23)
```

The reason is sometime root project does not define those values so it will fallback to our default values

Reply:
Yes, but react-native is pretty much unusable if you dont define rootProject.ext.compileSdkVersion and gradle/android does not work in 99% of the cases where you have mixed compileSdkVersions anyway.

Reply:
As a standalone library, we define all dependencies and its version to make sure everything run correctly with the the tested configuration.

Check this link for more information:
https://stackoverflow.com/questions/28371685/best-practice-for-buildscript-tag-in-build-gradle-of-library-projects

Reply:
We have updated the master branch based on your idea!

Reply:
perfect, thank you

Reply:
Soonish, we need to fix some other things first.